util/config.py

```diff
@@ -59,7 +53,22 @@ def initialize_globals():
     # Standard session configuration that'll be used for all new sessions.
     c.session_config = tf.ConfigProto(allow_soft_placement=True, log_device_placement=FLAGS.log_placement,
                                       inter_op_parallelism_threads=FLAGS.inter_op_parallelism_threads,
-                                      intra_op_parallelism_threads=FLAGS.intra_op_parallelism_threads)
+                                      intra_op_parallelism_threads=FLAGS.intra_op_parallelism_threads,
+                                      gpu_options=tf.GPUOptions(allow_growth=True))
```

Comment:
The `allow_growth` value here should be a flag rather than hardcoded to True, and it should default to False, our current behavior.

Reply:
> > > > which in turn block concurrent trainings.
> > > 
> > > 
> > > You are running several training on the same system ? How many GPUs do you have ?
> > 
> > 
> > Yes that was the case, I am training on a NVIDIA TITAN RTX.
> 
> Only one ? That feels counter-intuitive to overload just a single GPU for training

This is the resources available to me now and I want to fully utilise it, so why do you think this is counterintuitive?